src/Context/FeatureContext.php

```diff
@@ -929,6 +936,13 @@ public function build_phar( $version = 'same' ) {
 				$version
 			)
 		)->run_check();
+
+		// Revert the composer.json modification again
+		$this->composer_command( 'config --unset autoloader-suffix --working-dir=' . dirname( self::get_vendor_dir() ) );
+		// Prevent Composer from simply reusing the existing suffix.
+		$this->remove_dir( self::get_vendor_dir() . '/autoload.php' );
+		// Now it will use the content hash again.
+		$this->composer_command( 'dump-autoload --working-dir=' . dirname( self::get_vendor_dir() ) );
```

Comment:
Since this is using `vendor/bin/composer`, this fails, because without the `autoload.php` file one can't run Composer.

So I guess I can't delete the file after all. But if I don't do it, the hash is unchanged.

The alternative is for wp-cli/wp-cli-bundle to always have a fixed autoloader-suffix, like `WpCliBundle` or so. Then I could just reset to it.